src/hooks/number-validator/number-validator.test.jsx

```diff
@@ -0,0 +1,94 @@
+import { renderHook } from '@testing-library/react-hooks';
+import useNumberValidator from './number-validator.jsx';
+
+describe('useNumberValidator', () => {
+  it('passes validation for empty string', () => {
+    const { result } = renderHook(() => useNumberValidator(''));
+
+    expect(result.current).toBe(true);
+  });
+
+  it('passes validation for undefined', () => {
+    const { result } = renderHook(() => useNumberValidator(undefined));
+
+    expect(result.current).toBe(true);
+  });
+
+  it('passes validation for natural numbers', () => {
+    const { result } = renderHook(() => useNumberValidator('123'));
+
+    expect(result.current).toBe(true);
+  });
+
+  it('passes validation for rational numbers', () => {
+    const { result } = renderHook(() => useNumberValidator('0'));
```

Comment:
I think we're thinking of numbers like 1.5 or 3/2. But this might just be redundant since we already have a test for floating numbers below.

Reply:
Yeah, the tests are pretty thorough, so if you think they're good, I do too. :D